styles.css

```diff
@@ -554,4 +548,89 @@
     font-size: 15px;
     line-height: 20px;
   }
+
+  .contact-me {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 779px;
+    background: #6070ff;
+    background-image: url(./img/contact\ form\ shapes-mobile.svg);
+    background-position: 153px 0;
+    background-repeat: no-repeat;
+  }
+
+  .primary-text-contact-me {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #fff;
+    margin-top: 103px;
+  }
+
+  .secondary-text-contact {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+    margin-left: 24px;
+    margin-right: 24px;
+    margin-bottom: 35px;
+  }
+
+  /* Contac Me */
+
+  .form-contact-me {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .input {
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    height: 48px;
+    width: 327px;
+    margin-bottom: 20px;
+    padding-left: 15px;
+  }
+
+  #message {
+    height: 170px;
+    width: 327px;
+    padding-top: 10px;
+  }
+
+  .submit-button {
```

Comment:
:red_circle: Please make sure your submission button is positionned at the center as it is on the Figma design.
|Your Design|Figma Design|
|:---------------:|:----------------:|
|![Screenshot from 2023-01-24 23-25-11](https://user-images.githubusercontent.com/26736582/214420456-59095eeb-9b58-4b4f-853e-81cb6004f693.png)|![Screenshot from 2023-01-24 23-14-16](https://user-images.githubusercontent.com/26736582/214417901-61f6b996-0324-445d-9069-76ee723494a4.png)|

Reply:
height removed from the body

Reply:
Thank you @mdg0410 for pointing to this comment. I apologize for this requested change.